submissions/TUstiugov/html-css-popup/style.css

```diff
@@ -0,0 +1,252 @@
+  font-size: 100%;
+  line-height: 1;
+  font-size: 14px;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+
+input, button, textarea {
+  font-family: inherit;
+}
+
+input::-ms-clear {
+  display: none;
+}
+
+button {
+  cursor: pointer;
+}
+
+button::-moz-focus-inner {
+  padding: 0;
+  border: 0;
+}
+
+a, a:visited {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: none;
+}
+
+ul li {
+  list-style: none;
+}
+
+img {
+  vertical-align: top;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-size: inherit;
+  font-weight: inherit;
+}
+
+/*--------------------*/
+/*===========================================================================*/
+body {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-size: 20px;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+img {
+  width: 100%;
+}
+
+.flex-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+/* HEADER ===================================================================*/
+.header {
+  padding: 20px;
+}
+.navigation__list {
+  justify-content: flex-end;
+}
+.navigation__item {
+  margin: 0 15px;
+}
+.navigation__link {
+  display: inline-block;
+  transform: scale(1);
+  transition: 0.3s;
+}
+.navigation__link:hover {
+  transform: scale(1.1);
+}
+.navigation__link:focus {
+  border: 2px solid blue;
+}
+.navigation__link_popup {
+  width: 24px;
+  height: 24px;
+  background: url("./images/popup-icons/popup-button.png") 0 0/100% no-repeat;
+}
+.navigation__link_bell {
+  width: 24px;
+  height: 24px;
+  background: url("./images/popup-icons/bell.png") 0 0/100% no-repeat;
+}
+.navigation__link_user {
+  width: 30px;
+  height: 30px;
+  background: url("./images/popup-icons/ava-l.png") 0 0/100% no-repeat;
+  border-radius: 50%;
+}
+
+/*===========================================================================*/
+/* POPUP ===================================================================*/
+.popup {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  opacity: 0;
+  visibility: hidden;
+  transition: 0.3s;
+}
+.popup:target {
+  opacity: 1;
+  visibility: visible;
+}
+.popup:target .popup__content {
+  opacity: 1;
+  transform: perspective(600px) translate(0, 0) rotate(0deg);
+}
+.popup__area {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+}
+.popup__body {
+  justify-content: flex-end;
+  min-height: 100%;
+  padding: 60px 20px 30px;
+  background: rgba(0, 0, 0, 0.5);
+}
+.popup__content {
+  position: relative;
+  z-index: 10;
+  width: 50%;
+  height: 80vh;
+  color: #000000;
+  background-color: #ffffff;
+  box-shadow: 0 0 10px 0 grey;
+  overflow-y: auto;
+  opacity: 0;
+  transition: 0.3s;
+  transform: perspective(600px) translate(0, -100%) rotateX(45deg);
+}
+.popup__container {
+  flex-wrap: wrap;
+  justify-content: space-around;
+  gap: 20px;
+  padding: 20px;
+}
+.popup__container > li {
```

Comment:
Avoid styling by HTML tags except you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

Reply:
Done.